### DISCUSSION TITLE
How to disable snippet?

### DISCUSSION THREAD

**Discussion by nicitaacom** [microsoft/vscode-discussions#1013] [Extension Development QnA] [answered]
![image](https://github.com/microsoft/vscode-discussions/assets/39565703/2af4545f-1746-4d1f-8a59-97ebe7d9713f)

how to disable this set snippet?

**Comment by gjsjohnmurray** [comment, marked as answer]
It doesn't look like you are **developing extensions** for VS Code, so please review the [READ FIRST](https://github.com/microsoft/vscode-discussions/discussions/5) post which is pinned at the top of this forum. It explains the purpose of the forum and gives links to alternative resources.

**Reply by nicitaacom** [reply]
![image](https://github.com/microsoft/vscode-discussions/assets/39565703/bcca1152-544e-42e6-8998-ce520c83651e)

Where should I ask this question?

**Comment by nicitaacom** [comment]
# Answer above its not an answer

# Solution - https://www.youtube.com/watch?v=sNnybLyJnQU&ab_channel=Code2020
